contributing/release-schedule.md

```diff
@@ -1,84 +1,154 @@
+
+> Note: The LTS release channel is available only to AMP HTML pages; it is not available to AMP Actions, AMP for Email, AMP Stories, or AMP for Ads.
+
+```html
+<!-- Standard AMP HTML runtime and extension scripts -->
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+<script async custom-element='amp-ad'
+     src='https://cdn.ampproject.org/v0/amp-ad-0.1.js'></script>
+
+<!-- LTS AMP HTML runtime and extension scripts -->
+<script async src="https://cdn.ampproject.org/lts/v0.js"></script>
+<script async custom-element='amp-ad'
+     src='https://cdn.ampproject.org/lts/v0/amp-ad-0.1.js'></script>
+```
+
+As long as all runtime and extension scripts in a page must use the same release version, requesting the LTS scripts is considered valid AMP and gets the same cache benefits as the **stable** release channel.
+
+One trade-off to be aware of is that, because the **lts** release is updated every four weeks, the release binaries may be as much as six weeks behind the `HEAD` of [`ampproject/amphtml`](https://github.com/ampproject/amphtml). For this reason, partners using the **lts** release channel should not use very newly introduced features. See the section on [determining if your change is in a release](#Determining-if-your-change-is-in-a-release).
+
+## Determining if your change is in a release
+
+[_Type: Release_ GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) are used to track the status of current and past releases (from the initial cut to **experimantal**/**beta** testing to **stable**). Announcements about releases are made on the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
+
+You can determine what changes are in a given build using one of the following:
+
+- The [_Type: Release_ GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) for each release build will include a link to the specific [release page](https://github.com/ampproject/amphtml/releases) listing the PRs in that release.
+  - The most recent _Type: Release_ issue with _(Production)_ in the title tracks the build currently in the **stable** release.
+  - The _Type: Release_ issue with _(Canary)_ in the title tracks the build currently in the **experimental**/**beta** releases.
+- The [_PR Use: In Canary_](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Canary%22), [_PR Use: In Production_](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Production%22), and [_PR Use: In LTS_](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20LTS%22) labels are added to PRs when they've made it into a _weekly_ or **lts** build. There may be a delay between when the build is created and when the label is added.
```

Comment:
Worth mentioning that these labels are using legacy names, and that we'll be renaming the labels soon

Reply:
Donezo.